xwalk_android.gypi

```diff
@@ -22,6 +22,7 @@
       ],
       'include_dirs': [
         '..',
+        '../third_party/mojo/src',
```

Comment:
We usually don't need to change `include_dirs` like this, since most of the time the targets we depend on in `dependencies` already add whatever directories are required for them to work. Don't we need to depend on some mojo target instead?